wasm/src/language_service.rs

```diff
@@ -164,6 +164,7 @@ impl LanguageService {
                         qsls::protocol::CompletionItemKind::Variable => "variable",
                         qsls::protocol::CompletionItemKind::TypeParameter => "typeParameter",
                         qsls::protocol::CompletionItemKind::Field => "field",
+                        qsls::protocol::CompletionItemKind::Class => "class",
```

Comment:
Oh I almost forgot. You need to handle this enum case in both the vscode and playground and map them to the corresponding enum values in the VS Code and Monaco APIs. (There should be two changes, under `vscode/` and `playground/`). That way the proper icon will show up in the UI.